lib/filesystem/src/filesystem.cpp

```diff
@@ -88,10 +88,10 @@ namespace irods::experimental::filesystem::NAMESPACE_IMPL
             return ec;
         }
 
-        const auto rsRmColl = [](rsComm_t* _comm, collInp_t* _rmCollInp, int) -> int
+        const auto rsRmColl = [](rsComm_t* _comm, collInp_t* _rmCollInp, int p) -> int
         {
             collOprStat_t* stat{};
-            return ::rsRmColl(_comm, _rmCollInp, &stat);
+            return ::rsRmColl(_comm, _rmCollInp, p ? &stat : nullptr);
```

Comment:
what is `p` here?
should `p` be a boolean instead?

also missing leading underscore.

Reply:
p -> progress